OverlayPlugin.Core/NetworkProcessors/LineEventStartPlayFinish.cs

```diff
@@ -0,0 +1,436 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using RainbowMage.OverlayPlugin.NetworkProcessors.PacketHelper;
+
+/*
+Example lines from fishing:
+cast line was done before capturing.
+exclamation point animation:
+275|2024-05-12T00:43:34.1040000-04:00|1|10001234|0|150001|5|40001|0|1|125|4927e2b6b7d6d118
+reeling animation:
+275|2024-05-12T00:43:34.8510000-04:00|4|10001234|0|150001|6|40001|0|4|11C|120|10F|0|b0042cd36facf644
+finish reeling animation:
+275|2024-05-12T00:43:44.2490000-04:00|1|10001234|0|150001|2|40001|0|2|0|359f9119b64af7fd
+capture animation:
+275|2024-05-12T00:43:44.5690000-04:00|8|10001234|0|150001|4|40001|0|5|10F|0|1|1|7|0|0|0|157dc732c1530126
+ */
+namespace RainbowMage.OverlayPlugin.NetworkProcessors
+{
+    class LineEventStartPlayFinish : LineBaseCustom<
+            Server_MessageHeader_Global, LineEventStartPlayFinish.EventPlay1,
+            Server_MessageHeader_CN, LineEventStartPlayFinish.EventPlay1,
+            Server_MessageHeader_KR, LineEventStartPlayFinish.EventPlay1>
+    {
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        internal unsafe struct EventPlayBase
+        {
+            uint actorID;
+            uint unknown1;
+            uint eventID;
+            ushort scene;
+            ushort padding;
+            uint flags;
+            uint param3;
+            byte param4;
```

Comment:
Any particular reason these are `param3` and `param4` as opposed to `1` and `2`?  (ClientStructs thing?)  It just caught my eye as a little odd that we start the numbering at 3, but start at 1 for `EventStart`/`EventFinish`.  (Yes, this is internal and probably immaterial since we can relabel the fields in `netlog_defs`, so mostly just curious - not necessarily a change request.

Reply:
It starts at `3` because originally `scene/padding` were `param1`, and `flags` was `param2`. The joys of trying to figure these things out.